bin/llvm-krun

```diff
@@ -156,8 +161,18 @@ do
     shift; shift
     ;;
 
+    -vvv)
+    verbose+=3
```

Comment:
This line doesn't do what you think it does; this will concatenate the string `3` onto the _string_ value currently stored in `verbose`. For example:

**`test.sh`**
```bash
#!/usr/bin/env bash
verbose=0
verbose+=1
echo $verbose
```

```console
$ ./test.sh
01
```

The correct solution is to use shell [arithmetic expansion](https://tldp.org/LDP/abs/html/arithexp.html)

Reply:
My bad, so sorry!